blocks/customers/customers.css

```diff
@@ -0,0 +1,100 @@
+.customers{
+    margin-top: 20px;
+    margin-bottom: 60px;
+    padding:0 10px;
+}
+.customers__image{
+    position: relative;
+    background-image: url('../../images/forCustomers_mobile.png');
+    background-repeat: no-repeat;
+    background-size: cover;
+    min-height:643px ;
+    border-radius: 20px;
+}
+.customers__content{
+    position: absolute;
+    inset: 0;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    background-color: rgba(0, 0, 0, .5);
+    height: 100%;
+    border-radius: 20px;
+    
+}
+.customers__heading{
+    color: #fff;
+    font-size: 34px;
+    font-weight: 700;
+    padding: 36px 16px;
+}
+.customers__sub-content{
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    
+}
+.customers__text{
+    color: #fff;
+    font-family: 'Roboto' Arial, Helvetica, sans-serif;
+    font-weight: 400;
+    font-size: 18px;
+    margin: 20px 16px;
+    max-width: 584px;
+
+}
+.customers__button{
+    display: block;
+    color: #683D83;
+    font-size: 24px;
+    font-family: 'Roboto Condensed', Arial, Helvetica, sans-serif;
+    font-weight: 700;
+    line-height: 28px;
+    text-transform: uppercase;
+    margin: 20px 16px;
+    padding: 16px 69px;
+    background-color: #fff;
+    border-radius: 10px;
+    transition: ease-in-out .5s all;
+    border: none;
+}
+.customers__button:hover{
```

Comment:
There don't appear to be hover styles on mobile screens for these buttons in the design.